### PR TITLE
feat: Display GFN games from Firestore

### PR DESCRIPTION
**Pull request by Akahadaka**
Now also includes the use of `status`, where anything the cron can't find again is marked as _ARCHIVED_:
image

### CODE REVIEW

File: src/services/gfn.service.ts [hunk 1]
```diff
@@ -53,33 +46,23 @@ export interface ResponseError {
...
-            ...game,
-            appid: game.steamUrl
-              ? Number(game.steamUrl.split('/').pop())
-              : null,
-            source: 'GFN',
-            free: game.genres.includes('Free To Play'),
-          } as Game;
-        });
-      }),
+    return this._getData$(query).pipe(
```
Comment (himohimo) [inline src/services/gfn.service.ts:67]: I like how you abstracted this away.

**Review by himohimo — COMMENTED** [review]
Were you able to build the projects?
I see this (though I believe the `firebase.config.json` was in the other `feature/gfn-firestore `branch ):
```
 ERROR  Failed to compile with 1 errors                                                                                                                 8:17:37 PM

This relative module was not found:

* ./firebase.config.json in ./src/firebase.ts
ERROR in /Users/himohimo/Code/gfn-games/src/firebase.ts(4,28):
4:28 Cannot find module './firebase.config.json' or its corresponding type declarations.
    2 | import 'firebase/firestore';
    3 | import 'firebase/auth';
  > 4 | import firebaseConfig from './firebase.config.json';
      |                            ^
    5 | 
    6 | firebase.initializeApp(firebaseConfig);
    7 | 
Version: typescript 3.9.5
Time: 3206ms
```

**Comment by Akahadaka** [comment]
Yeah, sorry. I should've added a note somewhere. I got a complaint from github that I was trying to write a Google API Key to a public repo. Fair enough. It's probably still in the history of this branch, maybe I'll squash and merge, but I moved it to the file `/src/firebase.config.json` which is .gitignored.

You can fill the contents of that json file from Firebase Project > Settings > General: Scroll down to the bottom can copy the contents of the Config. You'll need to format it for json. i.e. 
```json
{
    "apiKey": "GOOGLEFIREBASEAPIKEY",
    "authDomain": ""
}
```

File: src/services/gfn.service.ts (hunk 1 above)
Comment (Akahadaka) [reply src/services/gfn.service.ts:67]: Yeah. With AngularFire you just do `query.valueChanges()` and get an observable returned. I see there is a `vue-firestore` npm package, but this is all we need for now.

**Comment by Akahadaka** [comment]
Irresponsibly adding more to this diff
**NOTE:** This branch is currently deployed
image

**Pull request merged**